CHANGELOG.md

```diff
@@ -34,6 +34,7 @@ and the versioning aims to respect [Semantic Versioning](http://semver.org/spec/
 - is traded at, trades (#1912)
 - replace term tracker item with term tracker annotation (#1922, #1923)
 - replace has bearer with characteristic of (#1928)
+- replace term tracker item with annotation in pull request template (#1938)
```

Comment:
```suggestion
```
The PR template is not an actual part of the OEO, therefore we don't need a CHANGELOG entry.

Reply:
Alright, I removed it.